### PR TITLE
Added authored and workflow-stage filter to article teaser-selection

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | requires https://github.com/sulu/sulu/pull/3464
| License | MIT

#### What's in this PR?

This PR will add the filter list to the overlay selections.

### CODE REVIEW

File: Resources/public/js/components/article-selection/main.js [hunk 1]
```diff
@@ -103,7 +103,7 @@ define([
                         }.bind(this),
                         selectedCounter: true,
                         searchInstanceName: this.options.instanceName + '-article-search',
-                        searchFields: ['id', 'title'],
+                        searchFields: ['title', 'route_path', 'changer_full_name', 'creator_full_name', 'pages.title'],
```
Comment (alexander-schranz) [inline Resources/public/js/components/article-selection/main.js]: author?

File: composer.json [hunk 2]
```diff
@@ -5,7 +5,7 @@
     "license": "MIT",
     "require": {
         "php": "^5.5 || ^7.0",
-        "sulu/sulu": "^1.6.1",
+        "sulu/sulu": "dev-master as 1.6.3",
```
Comment (alexander-schranz) [inline composer.json]: should be `^1.6.3` or `^1.6.3 || dev-master`

File: Resources/public/js/components/article-selection/main.js (hunk 1 above)
Comment (alexander-schranz) [inline Resources/public/js/components/article-selection/main.js]: and do we really want to be searchable for `pages.title`?

File: composer.json (hunk 2 above)
Comment (wachterjohannes) [inline composer.json]: when release 0.7 we have to change that to `^1.6.3` but now there is no 1.6.3 so this should be good.

**Comment by wachterjohannes** [comment]
@alexander-schranz fixed

**Pull request merged**